wbooks/src/app/index.js

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 
 import './styles.css';
 import Dashboard from './screens/dashboard';
+import Login from './screens/login'
 
 const App = () =>
   <div className="App">
-    <Dashboard />
+    <Login />
```

Comment:
It's desirable to mock this behavior - once the login button is clicked, this should redirect to `Dashboard`